src/span.js

```diff
@@ -155,35 +160,50 @@ export default class Span {
      *
      * @param {Object} keyValuePairs - An object with key value pairs
      * that represent tags to be added to this span.
+     * @return {Span} - returns this span.
      **/
-    addTags(keyValuePairs: any): void {
-        if (this._spanContext.isSampled()) {
-            for (let key in keyValuePairs) {
-                if (keyValuePairs.hasOwnProperty(key)) {
-                    let value = keyValuePairs[key];
-                    this._tags.push({key, value});
-                }
+    addTags(keyValuePairs: any): Span {
+        for (let key in keyValuePairs) {
+            if (keyValuePairs.hasOwnProperty(key)) {
+                let value = keyValuePairs[key];
+                this.setTag(key, value);
```

Comment:
this is not very efficient, I would append to _tags directly here, or use a helper method _setTag which does not do the check for isSampled over and over